pybaseball/team_results.py

```diff
@@ -69,7 +69,7 @@ def get_table(soup: BeautifulSoup, team: str) -> pd.DataFrame:
     df = pd.DataFrame(data)
     df = df.rename(columns=df.iloc[0])
     df = df.reindex(df.index.drop(0))
-    df = df.drop('',1) #not a useful column
+    df = df.drop('',axis=1) #not a useful column
```

Comment:
LGTM, but nit: space?

Reply:
Lol leave it to me to mess up a 5 character commit.